spaces/S000065/properties/P000089.md

```diff
@@ -0,0 +1,15 @@
+---
+space: S000065
+property: P000089
+value: false
+---
+
+The map
+$$f(x)=
+\begin{cases}
+1^* & x = 1\\
+1 & x = 1^*\\
+\frac{1 + x}{2} & \text{otherwise}
```

Comment:
```suggestion
The map $f:X\to X$ defined by
$$f(x)=
\begin{cases}
1^* &\text{if }x = 1,\\
1 &\text{if }x = 1^*,\\
(1 + x)/2 & \text{otherwise}
```